control-station/src/components/SensorBoxes/Camera.tsx

```diff
@@ -1,7 +1,29 @@
 import "./SensorData.css";
+import { LineChart } from "@mui/x-charts";
+import PodContext from "@/services/PodContext";
+import { useContext, useEffect, useState } from "react";
 
 function Camera() {
-	return <div className="camera"></div>;
+	const { podData } = useContext(PodContext);
+	const [lidarList, setLidarList] = useState<number[]>([]);
+
+	useEffect(() => {
+		console.log(lidarList);
+		setLidarList((prevLidarList) => [...prevLidarList, podData.lidar]);
+	}, [podData, lidarList]); // Added lidarList to the dependency array
```

Comment:
Chore: As mentioned before, please avoid including descriptions of changes as comments. If you want to record this information, you can elaborate in the body of the commit message.